app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -3,8 +3,56 @@ kind: registration
 name: Global Registration
 model: ProvisioningTemplate
 -%>
+#!/bin/sh
+<%
+    headers = ["-H 'Content-Type: application/json'", "-H 'Accept: application/json'", "-H 'Authorization: Bearer #{@auth_token}'"]
+-%>
+
+# Rendered with following template parameters:
+<%= "#  - User: [#{@user.login}]" if @user %>
+<%= "#  - Organization: [#{@organization.name}]" if @organization %>
+<%= "#  - Location: [#{@location.name}]" if @location %>
+<%= "#  - Hostgroup: [#{@hostgroup.name}]" if @hostgroup %>
 
 if ! [ $(id -u) = 0 ]; then
     echo "Please run as root"
     exit 1
 fi
+
+if [ -f /etc/os-release ] ; then
+  . /etc/os-release
+else
+    echo "Missing /etc/os-release file!"
+    exit 1
```

Comment:
This is not what I intended. IMHO it should deal with a missing `/etc/os-release`. AFAIK we still want to support EL6 for example, which doesn't have it. Also non-Linux unix installs won't have it. Or really all non-systemd.

Reply:
Well what would you suggest? IMHO this should be up to users changes,
that is why we used templates for the feature, so users could edit whatever they want.

Reply:
But you can easily fix it to support it. In fact, in the non-katello case you don't even use variables from this file.